server/user/userController.js

```diff
@@ -276,7 +276,7 @@ module.exports = {
               console.log(error);
               res.status(400).send(error);
             } else {
-              res.send(201).send(updatedUser);
```

Comment:
just for code consistency-- all the other functions call this user object returned from mongoose `user`, but this one calls it `updatedUser`.